app/javascript/packages/document-capture/components/document-capture.tsx

```diff
@@ -137,12 +138,15 @@ function DocumentCapture({ onStepChange = () => {} }: DocumentCaptureProps) {
 
   // If the user got here by opting-in to in-person proofing, when skipDocAuth === true,
   // then set steps to inPersonSteps
-  const steps: FormStep[] = skipDocAuth ? inPersonSteps : defaultSteps;
+  const steps: FormStep[] = skipDocAuth || skipDocAuthFromHandoff ? inPersonSteps : defaultSteps;
```

Comment:
I've gradually become a fan of trying to rewrite anything I have to stop and think about for a minute, not so much because I'm intellectually lazy, but because I feel like those are the places where bugs can hide.

How would you feel about adding (technically unnecessary) parenthesis to this line to make explicit the order of operations? I think this assignment could be read as either:

```js
(skipDocAuth || skipDocAuthFromHandoff) ? inPersonSteps : defaultSteps;
```

or

```js
skipDocAuth || (skipDocAuthFromHandoff ? inPersonSteps : defaultSteps);
```

I'm pretty sure I know which one is correct, but I think we could make it so obvious that it's immediately obvious to everyone.

Reply:
@n1zyy funny if I use `(skipDocAuth || skipDocAuthFromHandoff) ? inPersonSteps : defaultSteps;`, the prettier formatter seems automatically removed it parentheses. 

I changed to
```typescript
const isInPersonStepEnabled = skipDocAuth || skipDocAuthFromHandoff;
const steps: FormStep[] = isInPersonStepEnabled ? inPersonSteps : defaultSteps;
```